enzyme/Enzyme/Utils.cpp

```diff
@@ -655,8 +655,7 @@ void callMemcpyStridedBlas(llvm::IRBuilder<> &B, llvm::Module &M, BlasInfo blas,
 void callMemcpyStridedLapack(llvm::IRBuilder<> &B, llvm::Module &M,
                              BlasInfo blas, llvm::ArrayRef<llvm::Value *> args,
                              llvm::ArrayRef<llvm::OperandBundleDef> bundles) {
-  std::string copy_name =
-      (blas.prefix + blas.floatType + "lacpy" + blas.suffix).str();
+  std::string copy_name = blas.prefix + blas.floatType + "lacpy" + blas.suffix;
```

Comment:
unnecessary heap allocations for construction temporary std::strings!
https://godbolt.org/z/Ecdcc5beK